src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +26,7 @@
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Data successfully fetched from DB");
```

Comment:
The log message 'Data successfully fetched from DB' is misleading because the data is actually being created in the method and not fetched from a database. Consider updating the log message to reflect the actual operation.